shopify-client/src/components/ShopForm.vue

```diff
@@ -41,19 +41,126 @@
 
       <b-form-group
         description="Pick an image for your shop"
-        id="input-group-5"
-        label-for="input-5"
+        id="input-group-3"
+        label-for="input-3"
         label="Shop Image:"
       >
         <b-form-file
           :state="Boolean(form.image)"
           accept=".png"
           drop-placeholder="Drop image here..."
-          id="input-5"
+          id="input-3"
           placeholder="Choose an image or drop it here..."
           v-model="form.image"
         ></b-form-file>
       </b-form-group>
+
+      <b-form-group
+              description="Shop Tags"
+              id="input-group-4"
+              label-for="input-4"
+              label="Shop Tags:"
+      >
+        <div v-if="form.tags.length >= 1">
+          <h4>
+            <p v-for="tag in form.tags"
+               v-bind:key="'form.tag' + tag.id">
+                <b-badge variant="info" small
+                         :for="tag.name">{{tag.name}}
+                  <b-button class="ml-1" variant="danger" size="sm"
+                            v-on:click = "removeTag(tag.name)" title="Remove Tag">
+                    X
+                  </b-button>
+                </b-badge>
+            </p>
+          </h4>
+        </div>
+      </b-form-group>
+
+      <b-form-group
+              description="Add New Tags"
+              id="input-group-5"
+              label-for="input-5"
+              label="Add Tag:"
+      >
+        <div v-if="tags.length >= 1">
+          <label class="mr-2">Existing Tags: </label>
+          <select id="addTagSelect">
+            <option disabled value="">Select Tag</option>
+            <option  v-for="tag in tags"
+                     v-bind:key="'tag' + tag.id"
+                     :for="tag.name" value="tag.id">
+              {{tag.name}}
+            </option>
+          </select>
+          <b-button class="ml-2" variant="primary" size="sm" title="Add Tag"
```

Comment:
nitpicking again, these attributes should be on new lines

Reply:
3fd4fa0